contrib/deterministic-build/requirements-hw.txt

```diff
@@ -220,6 +220,22 @@ pyaes==1.6.1 \
 pycparser==2.21 \
     --hash=sha256:8ee45429555515e1f6b185e78100aea234072576aa43ab53aefcae078162fca9 \
     --hash=sha256:e644fdec12f7872f86c58ff790da456218b10f863970249516d60a5eaca77206
+pysatochip==0.12.5 \
```

Comment:
Why is 0.14.1 and 0.14.2 marked as latest on pypi (but has a date of 2021)? That confused me.